frontend/src/scenes/error-tracking/ErrorTrackingGroupScene.tsx

```diff
@@ -30,8 +31,16 @@ const STATUS_LABEL: Record<ErrorTrackingGroup['status'], string> = {
 }
 
 export function ErrorTrackingGroupScene(): JSX.Element {
-    const { group } = useValues(errorTrackingGroupSceneLogic)
-    const { updateGroup } = useActions(errorTrackingGroupSceneLogic)
+    const { group, groupLoading } = useValues(errorTrackingGroupSceneLogic)
+    const { updateGroup, loadGroup } = useActions(errorTrackingGroupSceneLogic)
+
+    useEffect(() => {
+        // don't like doing this but scene logics do not unmount after being loaded
+        // so this refreshes the group on each page visit in case any changes occurred
+        if (!groupLoading) {
+            loadGroup()
+        }
+    }, [])
```

Comment:
kea has an onpropschange might be able to use that instead?

Reply:
e.g. https://github.com/PostHog/posthog/blob/69becac2009bb4d17fcb2a5bd90d2a2b9219873a/frontend/src/lib/components/HTMLElementsDisplay/htmlElementsDisplayLogic.ts#L47

Reply:
but also happy to have this in there so in 6 months we can say "oh, that's why we should have fixed that" 😅

Reply:
This isn't so much about visiting a different group, it's needed even if you go back to the same group. Scenario is:

1. You're on the group page and click the breadcrumb back to the index page
2. You update the assignee of the group on the index page
3. You visit the group again but the old assignee is still there

Because the scene logic isn't unmounted in 1. (I don't understand why fully, I would have hoped / expected it was) when you come back to the group in 3. it was using the original value unless we reload it like so